sdk/bpf/scripts/install.sh

```diff
@@ -128,57 +129,36 @@ if [[ ! -e criterion-$version.md || ! -e criterion ]]; then
   touch criterion-$version.md
 fi
 
-# Install LLVM
-version=v0.0.15
-if [[ ! -e llvm-native-$version.md || ! -e llvm-native ]]; then
-  (
-    set -e
-    rm -rf llvm-native*
-    rm -rf xargo
-    job="download \
-           https://github.com/solana-labs/llvm-builder/releases/download \
-           $version \
-           solana-llvm-$machine.tar.bz2 \
-           llvm-native"
-    get $version llvm-native "$job"
-  )
-  exitcode=$?
-  if [[ $exitcode -ne 0 ]]; then
-    exit 1
-  fi
-  touch llvm-native-$version.md
-fi
-
 # Install Rust-BPF
-version=v0.2.5
-if [[ ! -e rust-bpf-$machine-$version.md || ! -e rust-bpf-$machine ]]; then
+version=v1.0
```

Comment:
why not use the same versioning as rust, i.e. v1.46?

Reply:
because we might make changes to the toolchain on our end that might necessitate v1.x,

Reply:
1.46.x then maybe?

Reply:
We could, thought that also might conflict with a rust version